src/Arcus.Security.Providers.HashiCorp/VaultKeyValueSecretEngineVersion.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Arcus.Security.Providers.HashiCorp
+{
+    /// <summary>
+    /// Represents the API version of the client engines when interacting with the HashiCorp Vault in the <see cref="HashiCorpSecretProvider"/>.
```

Comment:
Maybe link to the official docs for this?

Reply:
Yes, was thinking maybe we should provide more links to the HashiCorp Vault docs throughout the source code and docs.